page.yaml

```
@@ -593,6 +593,16 @@ groups:
           exclude example.com/old/thing v1.2.3
           replace example.com/bad/thing v1.4.5 => example.com/good/thing v1.4.5
           retract [v1.9.0, v1.9.5]
+      - title: Enforce Go code architecture with `go-arch-lint`
```

you don't need to put name like this. `with <name>` is added automatically
```suggestion
      - title: Enforce Go code architecture
```